fusor-ember-cli/tests/unit/utils/validators-test.js

```diff
@@ -308,6 +309,35 @@ test('IpAddressValidator rejects invalid values', function (assert) {
   });
 });
 
+test('IpTypoValidator accepts valid values of 3 or more characters', function (assert) {
+  let ipTypoValidator = IpTypoValidator.create({});
+  let validValues = [
+    'abc.168.2.0',
+    'a192.b168.c153.0',
+  ];
+
+  validValues.forEach((value) => {
+    assert.ok(ipTypoValidator.isValid(value), `"${value}" was not accepted as valid`);
+    assert.notOk(ipTypoValidator.isInvalid(value), `"${value}" was rejected as invalid`);
+    assert.equal(ipTypoValidator.getMessages(value).length, 0);
+  });
+});
+
+test('IpTypoValidator rejects invalid values 2 or less characters', function (assert) {
+  let ipTypoValidator = IpTypoValidator.create({});
+  let invalidValues = [
+    '192.168.2.1a',
+    '192a.1b62.257.1',
+    'ab192.168.1.2',
```

Comment:
@jwmatthews, the [RFC](https://tools.ietf.org/html/rfc1123#page-13) doesn't specify 2 or less characters, but we can discuss this. I assumed that these are typos
```
192.168.2.1a
192a.1b62.257.1
ab192.168.1.2
```
but if a user types in 3 characters or more, then it's not longer a typo, like these
```
abc.168.2.0
a192.b168.c153.0
```
seems strange to have hostnames like this, but what is the other alternative to catching typos?